plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/schema/SQLSchemaManager.java

```diff
@@ -51,22 +51,22 @@ public final class SQLSchemaManager {
     private final SQLSchemaConnectionProvider connectionProvider;
     private final SQLSchemaVersionManager versionManager;
 
-    private SQLDialect targetDatabaseDialect;
-    private String targetDatabaseName;
+    private final SQLDialect targetDatabaseDialect;
+    private final String targetDatabaseName;
 
-    private int schemaVersionActual;
-    private int schemaVersionObsolete;
+    private final int schemaVersionActual;
+    private final int schemaVersionObsolete;
 
     public SQLSchemaManager(
-        String schemaId,
-        SQLSchemaScriptSource scriptSource,
-        SQLSchemaConnectionProvider connectionProvider,
-        SQLSchemaVersionManager versionManager,
-        SQLDialect targetDatabaseDialect,
-        String targetDatabaseName,
-        int schemaVersionActual,
-        int schemaVersionObsolete)
-    {
+            String schemaId,
+            SQLSchemaScriptSource scriptSource,
+            SQLSchemaConnectionProvider connectionProvider,
+            SQLSchemaVersionManager versionManager,
+            SQLDialect targetDatabaseDialect,
+            String targetDatabaseName,
+            int schemaVersionActual,
+            int schemaVersionObsolete
```

Comment:
Please change the continuation indent from 8 spaces to 4 spaces.

In IntelliJ IDEA, it can be configured in <kbd>Editor</kbd> &rArr; <kbd>Code Style</kbd> &rArr; <kbd>Java</kbd> &rArr; <kbd>Tabs and Indents</kbd> &rArr; <kbd>Continuation indent</kbd>.